docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -11,10 +11,7 @@ onto a CHIP-enabled Thread network. The instructions are also valid for
 
 -   [Overview](#overview)
 -   [Requirements](#requirements)
--   [Building and programming OpenThread RCP firmware](#building-rcp-firmware)
--   [Configuring PC as Thread Border Router](#configuring-pc)
-    -   [Forming Thread network](#forming-thread-network)
-    -   [Configuring Wi-Fi hotspot](#configuring-hotspot)
+-   [Setting up Thread Border Router](#setting-up-thread-border-router)
 -   [Building and programming nRF Connect Lock Example Application](#building-example)
```

Comment:
What about changing Lock Example to Lighting Example or list them both (if not in the title, maybe in the section below).

Reply:
There is information at the beginning of the article that it can be adapted to the light example, too. I thought we wanted the article to be more specific and have less digressions. Why would the lighting example be better?

Reply:
That's fine, my thinking was that the current Android Application is better suited for lighting app, as it has also the scroll to control brightness. That's why I was considering making Lighting example primaty, but I'm fine either way.